README.md

```diff
@@ -49,90 +37,84 @@ app.use(errorHandler({
 app.listen(3000);
 ```
 
-In LoopBack applications, add the following entry to your
-`server/middleware.json` file.
+In LoopBack applications, add the following entry to `server/middleware.json`:
 
 ```json
 {
   "final:after": {
     "strong-error-handler": {
       "params": {
          "debug": false,
-         "log": true,
+         "log": true
        }
     }
   }
 }
 ```
 
-## Content Type
+The above configuration will log errors to the server console, but not return stack traces in HTTP responses.
+For details on configuration options, see below.
 
-Depending on the request header's `Accepts`, response will be returned in
- the corresponding content-type, current supported types include:
-- JSON (`json`/`application/json`)
-- HTML (`html`/`text/html`)
+### Response format and content type
 
-*There are plans to support other formats such as Text and XML.*
+The `strong-error-handler` package supports HTML and JSON responses:
 
-## Options
-
-#### debug
-
-`boolean`, defaults to `false`.
+- When the object is a standard Error object, it returns the string provided by the stack property in HTML/text
+  responses.
+- When the object is a non-Error object, it returns the result of `util.inspect` in HTML/text responses.
+- For JSON responses, the result is an object with all enumerable properties from the object in the response.
 
-If you need to set the environment to development mode, you will need to change the value to `true`.
+The content type of the response depends on the request's `Accepts` header.
 
-When enabled, HTTP responses include all error properties, including
-sensitive data such as file paths, URLs and stack traces. *See Examples below*
+-  For Accepts header `json` or `application/json`, the response content type is JSON.
+-  For Accepts header `html` or `text/html`, the response content type is HTML.
 
-#### log
+*There are plans to support other formats such as Text and XML.*
 
-`boolean`, defaults to `true`.
+## Options
 
-When enabled, all errors are printed via `console.error`. That includes an array of fields (custom error properties)
- that are safe to include in response messages (both 4xx and 5xx).
+| Option | Type | Default | Description |
```

Comment:
I understand your arguments for using the table layout. On the other hand, most packages on npm use the narrative format, I think Node.js developers are accustomed to that. Formatting the argument documentation as a table may make it more difficult to read the content, simply because the form is not familiar.

> As far as linking to a specific property, this has never come up and no one has asked for it before. IMO it's sufficient to state the property and link to the "Options" section, which is what we do everywhere else.

Well, I have personally run into this issue many times, but never bothered to report it. Mae culpa. I cannot tell how much difficult it is for the readers when the link is in the form of URL + "and go find this property name", so maybe it doesn't really matter.